### PR TITLE
[Java] update feign form dependency version

### PR DESCRIPTION
**Pull request by gaetPax**
https://github.com/OpenAPITools/openapi-generator/issues/5173

In templates, updated feign and feign form dependencies versions to their latests when configOptions 'feignVersion' is set to '10.x'

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by gaetPax** [comment]
> Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/OpenAPITools/openapi-generator/graphs/contributors.
> 
> Let me know if you need help fixing it.
> 
> Ref: https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

Thank you, i fixed it. It should be okay now.

File: samples/client/petstore/java/feign10x/src/main/java/org/openapitools/client/auth/OAuth.java [hunk 1]
```diff
@@ -81,19 +81,19 @@ public void apply(RequestTemplate template) {
         }
         // If first time, get the token
         if (expirationTimeMillis == null || System.currentTimeMillis() >= expirationTimeMillis) {
-            updateAccessToken();
+            updateAccessToken(template);
```
Comment (wing328) [inline samples/client/petstore/java/feign10x/src/main/java/org/openapitools/client/auth/OAuth.java]: @gaetPax I think you will need to update the mustache template as well: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/Java/libraries/feign/auth/OAuth.mustache#L86

**Comment by wing328** [comment]
Bitrise CI failure not related to this PR.

**Comment by wing328** [comment]
cc @bbdouglas (2017/07) @sreeshas (2017/08) @jfiala (2017/08) @lukoyanov (2017/09) @cbornet (2017/09) @jeff9finger (2018/01) @karismann (2019/03) @Zomzog (2019/04) @lwlee2608 (2019/10) @bkabrda (2020/01)

**Comment by wing328** [comment]
@gaetPax thanks for the PR, which has been included in the 4.3.0 release: https://twitter.com/oas_generator/status/1243455743937789952

**Pull request merged**